WordPress/create-block-theme
# Update theme form markup and styles

This change adds a handful of improvements to the markup and styles for the main plugin page, including

- Radio button labels and form field descriptions match other wp-admin page style and structure more closely
- All description text is tied to its input field with an `aria-describedby` attribute
- Theme tags have responsive column styles
- URLs are removed from translation strings

### Testing instructions

- The changes should be markup structure and styles only
- Test theme creation at `/wp-admin/themes.php?page=create-block-theme` and make sure the form submits successfully

| **Before** | **After** |
| - | - |
| ![image](https://github.com/WordPress/create-block-theme/assets/1699996/211e83f3-19c5-4c1e-8922-d94bbc8f43c6) | ![image](https://github.com/WordPress/create-block-theme/assets/1699996/636c78e6-6d1b-4e6c-9432-8a6c3e79edb3) |
| ![image](https://github.com/WordPress/create-block-theme/assets/1699996/a3494766-bab1-47ff-a9fa-43ae2c21953d) | ![image](https://github.com/WordPress/create-block-theme/assets/1699996/4bd5521d-fa5c-43a3-82b0-ad56813addaf) |
